### ISSUE TITLE
Document how to add Windows Terminal profile for a given Conda environment 

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#792]
Similar to https://dev.to/voodu/windows-terminal-conda-d3e, but now Windows Terminal permit to add new profile from GUI:

![document_terminal_conda](https://user-images.githubusercontent.com/1857049/121350762-544c5f00-c92b-11eb-99e5-9179aeed5b04.PNG)

**Comment by traversaro** [comment]
Interestingly, @sgiraz did not have the "Add new" section. I just installed Windows Terminal from Windows Store, and I have it:
![add_new_profile](https://user-images.githubusercontent.com/1857049/132026210-c5674f59-e890-43e5-9696-b93d12c700b5.PNG)

**Comment by sgiraz** [comment]
Yeah, I noticed that too. In my case, the last time, this option wasn't visible due to the small size of the window.

Said that, now I'm trying the new [Windows Terminal Preview](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n8g5rfz9xk3?activetab=pivot:overviewtab) which contains the latest features as they are developed.

For example, an useful option I found is the possibility to split the tab using context menu maintaining the current profile. Of course there have been introduced more other [interesting features](https://devblogs.microsoft.com/commandline/windows-terminal-preview-1-11-release/).

**Comment by traversaro** [comment]
Another useful trick to document is to define a profile that both activate a conda environent, *and* also calls the custom `setup.bat` scripts that needs to be called when the superbuild is compiled from source (see https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-forge.md#run-software-installed). Probably we can steal the same trick used in ROSOnWindows docs (see http://wiki.ros.org/Installation/Windows). The trick is to concatenate commands with `&&` .

**Comment by sgiraz** [comment]
Yep. Let me paste here the screen of the profile just for reference:

![image](https://user-images.githubusercontent.com/18039045/132170532-0bde8f15-db71-4a08-9a11-b437defc837d.png)

In general, the command has the following pattern:
`cmd.exe /k %HOME%/AppData/Local/mambaforge/Scripts/activate.bat <your-conda-env> && call <your robotology-superbuild-installation-path>/build/install/share/robotology-superbuild/setup.bat`

**Comment by traversaro** [comment]
According to @S-Dafarra this was something he was missing during one of the first actual tests of the conda-based Windows robotology-superbuild installation. In particular, for actual demos/deployments it would be also useful to also have specific desktop shortcuts that just open a specific Windows Terminal profile. For that the info in https://github.com/microsoft/terminal/issues/6366 could be useful.

**Comment by S-Dafarra** [comment]
Nice to know! I think we can start with the custom profile for the time being

**Comment by S-Dafarra** [comment]
Thanks @traversaro @sgiraz, I have added a custom profile in a Windows machine in our setup (``icub-virtualizer``). In our case the installation of ``mamba`` was in a different folder. Here the command I used inside the profile:
```
cmd.exe /k C:\mambaforge\Scripts\activate.bat robsub && call C:\robotology-superbuild\build\install\share\robotology-superbuild\setup.bat
```

I have also found a *yarpino* icon in ``YARP`` and used it as an icon.
![image](https://user-images.githubusercontent.com/18591940/141350051-c657547d-3452-49bf-80c3-8a0d43c9e514.png)


 Here how it looks when opening ``Windows Terminal``:
![image](https://user-images.githubusercontent.com/18591940/141349905-26d58049-581b-420e-b885-7b9d73ed26a4.png)


cc @isorrentino @RiccardoGrieco @kouroshD @lrapetti

**Comment by G-Cervettini** [comment]
@traversaro I'm struggling with an error while trying to install Windows Terminal on the new ALIENWARE IITICUBLAP290.
Do you know what could be going on here?
![image](https://user-images.githubusercontent.com/101020439/189683383-785e6771-5868-4958-828a-8ff1a1546c38.png)
